pkg/pillar/cmd/volumemgr/handlesnapshot.go

```diff
@@ -124,6 +124,8 @@ func handleVolumesSnapshotModify(ctxArg interface{}, key string, configArg, _ in
 		}
 		log.Noticef("handleVolumesSnapshotModify: successfully rolled back to snapshot %s for volume %s", config.SnapshotID, volumeID)
 	}
+	// Increment the refCount to indicate that the snapshot is being used and trigger the modify handler
+	volumesSnapshotStatus.RefCount++
```

Comment:
Probably I'm missing something, but where is the decrement?

Reply:
@rouming from the comment for RefCount it looks like this is a NumberTimesUsed and not a reference which needs to be decremented.